src/components/ConnectionsList.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useEffect, useState } from "react";
```

Comment:
I think we are toggling here? Maybe another name to consider is ToggleConnectionList